packages/cockpit/ui/src/reducers/selectors.js

```diff
@@ -176,7 +176,7 @@ export function getEnsErrors(state) {
 }
 
 export function isEnsEnabled(state) {
-  return Boolean(state.entities.plugins.find((plugin) => plugin.name === 'embark-ens'));
+  return Boolean(state.errorEntities.ensRecords !== "Request failed with status code 404");
```

Comment:
Is the plugin name never a truth or or always a truth? Asking, because maybe we want to test both conditions?
ie:
```
 return Boolean(state.entities.plugins.find((plugin) => plugin.name === 'embark-ens') && state.errorEntities.ensRecords !== "Request failed with status code 404");
```

Reply:
The plugin name for `embark-ens` is never truth, so I changed it to the path for the  `(plugin) => plugin.name === 'embark-ens' || plugin.name.includes("/namesystem/")`, but this was always truthful because the namesystem plugin is always loaded, but exits early if not enabled in the config.

So the easiest way forward to emit a request for a `.resolve()`, and see if the API was not loaded, which would produce a 404 for the request.

There are probably better alternatives, like an explicit endpoint for determining if ENS is indeed enabled.